src/spacingAliasesProvider.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import * as React from 'react'
+import PropTypes from 'prop-types'
+
+export type SpacingAliases = {
+  [spacingAlias: string]: number,
+}
+
+type Props = {
+  spacingAliases: SpacingAliases,
+  children: React$Node,
+}
+
+export const SpacingAliasesContext = { spacingAliases: PropTypes.object }
+
+export default class SpacingAliasesProvider extends React.Component<Props> {
```

Comment:
Should we just call this `Provider` (and export it as such) and use it as a single location for all configuration moving forward?

Reply:
What do you think about ConfigProvider?